### DISCUSSION TITLE
New build  and 4 PCBs for German / European Users

### DISCUSSION THREAD

**Discussion by bschymanski** [picocomputer/community#74] [Q&A]
Hi all, 

i just finished building my RP6502. I was fun and easy to put together and works fine!
Will ask questions in the near future

Now i am left with 4 PCBs that i could send to others from Germany or other places in Europe who would like to build their RP6502.

Cheers

Bernd

**Comment by Willbertz** [comment]
Hi Bernd!
I would be very interested. How much is it with post to Finland? Can you also tell me where did you buy all the rest of the parts?

Greetings!
Ville

**Comment by bschymanski** [comment]
Hi Ville,

sending a letter to Finnland is cheap, less than 2€.
You can send me your address to schymanski@gmail.com, i will send the pcb right away.

As for the parts: here: https://picocomputer.github.io/hardware.html under step 4, you can download a partslist. I already have an Account at Mouser, so i could just upload it and had to assign the fields for quantity and which field contains the mouser part number.
I removed the parts that i already had, like the Picos.
Best part: the bags for the parts contained the description what parts they are for: the 4.02K Resistors are for R2, R7, R12. Very easy!

Cheers

Bernd

**Comment by Willbertz** [comment]
Thank you so much Bernd!!

I sent you email.

All the best!
Ville
